backend/coins/btc/transaction_test.go

```diff
@@ -125,3 +142,82 @@ func TestGetFeePerKb(t *testing.T) {
 	}
 
 }
+
+func TestPickChangeAddressSucceeds(t *testing.T) {
+	rootFingerprint := []byte{1, 2, 3, 4}
+	signingConfigurations := signing.Configurations{
+		signing.NewBitcoinConfiguration(signing.ScriptTypeP2WPKHP2SH, rootFingerprint, mustKeypath(t, "m/49'/0'/1'"), test.TstMustXKey("xpub6CUmEcJb7juvpvNs2hKMc9BP1n82ixzUb4jyHUdYzSLmnXru3nb4hhGsfS23WRx8hgJLxMxZ7WcBGzTiYfiANUQZe3TVFghLrxvA2Ls7u4a")),
```

Comment:
Please try to keep lines below ~ 100 columns, i.e. by using one line per argument.

Reply:
Done, thanks